packages/react/src/toaster/Toaster.tsx

```diff
@@ -0,0 +1,32 @@
+import {
+  ToastProvider as RadixToastProvider,
+  ToastViewport as RadixToastViewport,
+} from "@radix-ui/react-toast";
+import { type ComponentPropsWithRef, forwardRef } from "react";
+
+import { Flex } from "../flex";
+import { Toast } from "../toast/Toast";
+import * as styles from "./Toaster.css";
+import { useToast } from "./use-toast";
+
+type ToasterProps = ComponentPropsWithRef<typeof RadixToastProvider> &
+  styles.ToastPositionVariants;
```

Comment:
let's use `BoxProps` here and extend Viewport
```suggestion
type ToastProps = BoxProps<
  typeof RadixToast.Viewport,
  {
    children?: ReactNode;
  } & ComponentPropsWithoutRef<typeof RadixToast.ToastProvider> &
    styles.ToastPositionVariants
>;
```